src/profile/external-identities/OAuth2Account.tsx

```diff
@@ -79,6 +80,7 @@ export const OAuth2Account = (props: OAuth2AccountProps) => {
         .ExternalCredentials(provider)
         .linkAccountWithAuthorizationCode(code, state);
       authStore.update(_.set(['oAuth2AccountStatus', provider.key], accountInfo));
+      Ajax().Metrics.captureEvent(Events.user.externalCredential.link, { provider: provider.key });
```

Comment:
provider.key isnt a secret or anything right?

Reply:
nope. Just a lowercase string of the provider, like `github` or `anvil`